CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.2.0
```

Comment:
```suggestion
## v0.1.0
```

Also delete the v0.1.0-dev section below.

Super minor, but the commit message should be `v0.1.0 release` for consistency with other projects.